src/Components/Main/Pages/NotFoundPage/NotFoundPage.tsx

```diff
@@ -1,9 +1,10 @@
+import type { FC, ReactElement } from 'react';
 import { useLocation } from 'react-router-dom';
 import { Error } from '../../../Common/Error/Error';
 
-export const NotFoundPage = () => {
+export const NotFoundPage: FC = (): ReactElement => {
   const { pathname } = useLocation();
-  const message = `The requested URL "${pathname}" was not found on this server!`;
+  const message: string = `The requested URL "${pathname}" was not found on this server!`;
```

Comment:
As I know, in such obvious string cases - it is not  required to set string type =)

Reply:
It is a redundant code. I have fixed it!